packages/tailwind/src/tailwind.spec.tsx

```diff
@@ -2,6 +2,7 @@ import { Tailwind } from "./tailwind";
 import { renderToStaticMarkup as render } from "react-dom/server";
 import { TailwindConfig } from "tw-to-css";
 import { Button } from "@react-email/button";
+import { Html, Head } from "@react-email/components";
```

Comment:
Here, can you use just the `<html/>` and `<head/>` to accomplish the same thing?. This way this module doesn't  have to be dependent on other modules.

Reply:
good idea, changed